### ISSUE TITLE
Unable to set status to idle using setStatusIdle() or setStatus(status, game, callback)

### ISSUE THREAD

**Issue by Jessassin** [hydrabolt/discord.js#134]
I have tried setStatusIdle() setStatusAway, and setStatus("idle") setStatus("away")

Using setStatus("idle","test") does set the game to "test" so it appears the issue is just with the idle status
```
    action: function(bot, msg) {
        bot.sendMessage(msg.channel, "Setting bot status to idle")
        bot.setStatus("idle", "test")
        bot.sendMessage(msg.channel, "Current bot status: " + bot.user.status) //returns "online"
        console.log("Status set to idle by: " + msg.sender.username)
      //}
    }
```

**Comment by abalabahaha** [comment]
@Jessassin If you can, please try c7b632d

**Comment by Jessassin** [comment]
Don't have my env set up to be able to test that. Will test on next release if that's okay

**Comment by Jessassin** [comment]
Just confirming that this was fixed after updating to release 6.0.0
